src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/resource/VisitSessionTemplateResource.kt

```diff
@@ -19,14 +19,14 @@ import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.visitscheduler.config.ErrorResponse
 import uk.gov.justice.digital.hmpps.visitscheduler.data.CreateSessionTemplateRequest
 import uk.gov.justice.digital.hmpps.visitscheduler.data.SessionTemplateDto
-import uk.gov.justice.digital.hmpps.visitscheduler.service.VisitSchedulerService
+import uk.gov.justice.digital.hmpps.visitscheduler.service.SessionTemplateService
 import javax.validation.Valid
 
 @RestController
 @Validated
-@RequestMapping(name = "Visit Resource", path = ["/visit-session-templates"], produces = [MediaType.APPLICATION_JSON_VALUE])
+@RequestMapping(name = "Session Resource", path = ["/visit-session-templates"], produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
may be Session Template Resource

Reply:
The session and Session Template resource are really the same, do you want me to update for consistency with the service names?